services/device-registry-file/src/main/java/org/eclipse/hono/deviceregistry/file/FileBasedCredentialsService.java

```diff
@@ -775,6 +786,7 @@ private CacheDirective getCacheDirective(final String type) {
             switch (type) {
             case CredentialsConstants.SECRETS_TYPE_HASHED_PASSWORD:
             case CredentialsConstants.SECRETS_TYPE_X509_CERT:
+            case CredentialsConstants.SECRETS_TYPE_PRESHARED_KEY:
```

Comment:
we deliberately chose to not allow caching of PSK credentials as they are plain text ...

Reply:
fixed